docs/privacy-policy.md

```diff
@@ -0,0 +1,33 @@
+# Privacy Policy
+
+*Effective 2020-10-05*
```

Comment:
Should we change this date to be more recent since we have been changing the policy?